docs/helpers.asciidoc

```diff
@@ -160,3 +160,79 @@ end
 scroll_helper.clear
 ----
 --
+
+[discrete]
+=== ES|QL Helper
+
+This helpers provides an Object response from the ESQL `query` API instead of the default JSON value.
+
+To use the ES|QL helper, require it in your code:
+
+[source,ruby]
+----
+require 'elasticsearch/helpers/esql_helper'
+----
+
+By default, the `query` API returns a Hash response with `columns` and `values` like so:
+
+[source,ruby]
+----
+query = <<ESQL
+        FROM sample_data
+        | EVAL duration_ms = ROUND(event.duration / 1000000.0, 1)
+ESQL
+
+response = client.esql.query(body: { query: query})
+puts response
+
+{"columns"=>[
+  {"name"=>"@timestamp", "type"=>"date"},
+  {"name"=>"client.ip", "type"=>"ip"},
+  {"name"=>"event.duration", "type"=>"long"},
+  {"name"=>"message", "type"=>"keyword"},
+  {"name"=>"duration_ms", "type"=>"double"}
+],
+"values"=>[
+  ["2023-10-23T12:15:03.360Z", "172.21.2.162", 3450233, "Connected to 10.1.0.3", 3.5],
+  ["2023-10-23T12:27:28.948Z", "172.21.2.113", 2764889, "Connected to 10.1.0.2", 2.8],
+  ["2023-10-23T13:33:34.937Z", "172.21.0.5", 1232382, "Disconnected", 1.2],
+  ["2023-10-23T13:51:54.732Z", "172.21.3.15", 725448, "Connection error", 0.7],
+  ["2023-10-23T13:52:55.015Z", "172.21.3.15", 8268153, "Connection error", 8.3],
+  ["2023-10-23T13:53:55.832Z", "172.21.3.15", 5033755, "Connection error", 5.0],
+  ["2023-10-23T13:55:01.543Z", "172.21.3.15", 1756467, "Connected to 10.1.0.1", 1.8]
+]}
+----
+
+The helper returns an Array of hashes with the columns as keys and the respective values. So for the previous example, it would return the following:
```

Comment:
```suggestion
The helper returns an array of hashes with the columns as keys and the respective values. So for the previous example, it would return the following:
```